### DISCUSSION TITLE
Field color and IF value

### DISCUSSION THREAD

**Discussion by IgorRSU** [YetiForceCompany/YetiForceCRM#14842] [How to]
Is it possible to set field color, for example, if field value is more then 10 to green and red to less 10 ?
Thank You

**Comment by mariuszkrzaczkowski** [comment]
there is no such possibility, but you can always add

**Reply by IgorRSU** [reply]
Thank You for quick answer. But where to start?
[PL] Dzięki za szybką odpowiedź. Od czego mam zacząć?

**Reply by mariuszkrzaczkowski** [reply]
It depends whether you want to do it on your own or you would like us to do it for you?
[PL] zależy czy robisz to sam czy zlecasz nam ?

**Reply by IgorRSU** [reply]
I will try to do it myself :)
[PL] Chciałbym spróbować zrobić to samodzielnie. Wdzięczny będę za podpowiedź z której strony mam się za to zabrać.

**Reply by alexgrndprx** [reply]
You can set conditions in *.tpl file.
For example you can add
`{if $LISTVIEW_HEADERNAME==='status' && $VALUE==="{\App\Language::translate('LBL_INACTIVE',$QUALIFIED_MODULE)}"}
<span style="background-color:#FF050563">
{/if}`
to layouts\basic\modules\Settings\CronTasks\ListViewContents.tpl so inactive tasks will have red background.
I'm not sure if it's correct way to do it, but it works in some cases.
